hardinfo.1

```diff
@@ -0,0 +1,58 @@
+.\" Manpage for hardinfo.
+.\" https://github.com/lpereira/hardinfo/.
+.TH HARDINFO "1" "July 2017" "0.6" "User Commands"
+.SH NAME
+Hardinfo \- System profiler and benchmark for Linux systems.
+.SH USAGE
+hardinfo [OPTION...]
+.SH DESCRIPTION
+HardInfo is a system profiler and benchmark for Linux systems.
+
+HardInfo can gather information about your operating system 
```

Comment:
I'd change this to:

    It gathers information about the hardware and operating system, performs benchmarks,
    and generates reports.  The information gathered is presented on an simple and intuitive
    GUI.

    Information is grouped in four main groups: Computer, Devices, Network, and Benchmarks.

    The Computer group contains mostly information about basic software.  This includes but
    is not limited to information about the operating system (kernel version, C library,
    distribution, etc.), kernel modules, locale information, filesystem usage, users/groups, and
    development tools.

    The Devices group contains information about the hardware, as detected by the operating
    system.  HardInfo is incapable of detecting hardware by itself, it merely lists what the
    operating system was able to find.  This includes, but does not limit to, information about the
    processor (including cache layout, known bugs, and feature flags with their explanation),
    devices (such as USB, PCI, and Input devices), among other things.

    The Network group lists information about network interfaces, routing table, DNS servers,
    and other networking-related topics.

    The Benchmarks group allow performing simple benchmarks to test CPU and FPU capabilities,
    as well as some of the graphical user interface capabilities as well.  In previous versions, it
    was possible to send the benchmark results to a central server, so it was possible to compare
    the benchmarks with other HardInfo users; this service has been deactivated in the mean time.

    Reports can be generated either by invoking HardInfo with special command-line parameters
    (see below), or by clicking the "Generate Report" button in the GUI.  Reports can be customized
    in the GUI and saved in either HTML or plain text formats.